### PR TITLE
helm: Make namespace configurable and CR opt out

### PR DESCRIPTION
**Pull request by ekarlso**

### CODE REVIEW

**Comment by tomaszsek** [comment]
Hi @ekarlso,

If you want to specify Jenkins CR namespace you have to configure watch namespace in the operator deployment. https://github.com/jenkinsci/kubernetes-operator/blob/930058d5b25f07b8996f88f021476f8de766e115/chart/jenkins-operator/templates/deployment.yaml#L36 Moreover RBAC have to adjusted too. Please look at https://github.com/jenkinsci/kubernetes-operator/blob/master/documentation/security.md#setup-jenkins-operator-and-jenkins-in-separated-namespaces.

Cheers

**Comment by ekarlso** [comment]
Yeah, I'll remove that part then but still we should have the CR be opt-outable

**Comment by pawelprazak** [comment]
In future I'd split formatting and actual changes for easier review, but apart from that, having the cr optional makes sense.

File: chart/jenkins-operator/templates/jenkins-cr.yaml [hunk 1]
```diff
@@ -1,6 +1,9 @@
+{{ if .Values.jenkins.enabled }}
 apiVersion: {{ .Values.jenkins.apiVersion }}
 kind: Jenkins
 metadata:
   name: {{ .Values.jenkins.name }}
+  namespace: {{ .Release.Namespace }}
```
Comment (pawelprazak) [inline chart/jenkins-operator/templates/jenkins-cr.yaml]: I believe @tomaszsek wanted this line removed

**Comment by tomaszsek** [comment]
@ekarlso New 0.0.3 version has been released.

**Pull request merged**